### ISSUE TITLE
Naming collisions in feature identifiers

### ISSUE THREAD

**Issue by romainmenke** [web-platform-dx/web-features#729]
I am working on mapping items from web-features onto items in cssdb and I can't help but notice how short and ambiguous the web-feature identifiers are.

For example the `:is()` pseudo class from CSS has only `is`.
But this could just as easily have been for the html attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/is

I am a bit worried that the current identifiers will prove to be too ambiguous and will be changed, requiring more work on my end to re-map all the features.

**Comment by foolip** [comment]
We are indeed picking short names and even shorter identifiers, the shortest possible really. This will eventually lead to identifiers we wish we could repurpose and probably inconsistencies. For example `scope` is taken by the `@scope` at-rule so the `:scope` pseudo-class will have to be called `scope-pseudo`. But the `:is()` functional pseudo-class is just `is`, not `is-pseudo`.

We have an `alias` field which we have used for the handful of identifier renames we've done so far. Once we have published web-features 1.0 I think we should make these guarantees:

- Removing an identifier is a breaking change and will bump the major version number.
- An `alias` is kept forever unless we really need to reuse it.
- Identifier reuse should be signaled some way ahead of time and also be a breaking change, when it's done.

I'm not totally sure the last step will ever be worth doing, or if we should just live with our mistakes.

Are there any additional guarantees cssdb needs to handle the situation?
